src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/README.md

```diff
@@ -86,10 +86,17 @@ if (err != 0)
 
 try
 {
+    //
+    // During development, it is useful to be able to validate against custom SSL
+    // certificates.  This logic would generally be removed in official releases.
```

Comment:
That explanation is a lot clearer and it would be good to include it in the comment.  And/or I'd suggest rewording the comment to clarify the context and the action you're asking for from the reader, e.g. "Include the following call if you want to allow entitlement tokens signed by a test certificate, passing the thumbprint of the certificate you used to sign your test tokens.  Remove it for production releases: this will ensure that the code only accepts tokens signed by the Azure Batch service."